frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -290,34 +290,29 @@ export const EnvironmentAccordionBody = ({
     };
 
     return (
-        <div>
-            <StyledAccordionBodyInnerContainer>
-                <StyledContentList>
-                    {releasePlans.length > 0 ? (
-                        <>
-                            {releasePlans.map((plan) => (
-                                <StyledListItem
-                                    type='release plan'
-                                    key={plan.id}
-                                >
-                                    <ReleasePlan
-                                        plan={plan}
-                                        environmentIsDisabled={isDisabled}
-                                    />
-                                </StyledListItem>
-                            ))}
-                            {strategies.length > 0 ? (
-                                <li>
-                                    <StrategySeparator />
-                                    <Strategies />
-                                </li>
-                            ) : null}
-                        </>
-                    ) : strategies.length > 0 ? (
-                        <Strategies />
-                    ) : null}
-                </StyledContentList>
-            </StyledAccordionBodyInnerContainer>
-        </div>
+        <StyledAccordionBodyInnerContainer>
+            <StyledContentList>
+                {releasePlans.length > 0 ? (
+                    <>
+                        {releasePlans.map((plan) => (
+                            <StyledListItem type='release plan' key={plan.id}>
+                                <ReleasePlan
+                                    plan={plan}
+                                    environmentIsDisabled={isDisabled}
+                                />
+                            </StyledListItem>
+                        ))}
+                        {strategies.length > 0 ? (
+                            <li>
+                                <StrategySeparator />
+                                <Strategies />
+                            </li>
+                        ) : null}
+                    </>
+                ) : strategies.length > 0 ? (
+                    <Strategies />
+                ) : null}
+            </StyledContentList>
+        </StyledAccordionBodyInnerContainer>
```

Comment:
Just removes the completely redundant wrapping div.

Reply:
I'm not sure about this structure. We have syntax that is very different from how it visually looks like. 
```
- ol
  - li // release plan
  - li
    - ol
      - li // strategy
      - li
      - li
```
It's because strategies are "draggable", right? 
And `<Strategies />` in-place component makes this harder to read.